src/sidebar/store/modules/session.js

```diff
@@ -52,6 +60,14 @@ function updateProfile(profile) {
   };
 }
 
+/**
+ *
+ * @returns {string}
```

Comment:
```suggestion
 * @return {string}
```

Both `@return` and `@returns` work, but `@return` is more de-facto conventional in our code.